### DISCUSSION TITLE
更新了预发行版后，3.3版本也打不开了……

### DISCUSSION THREAD

**Discussion by byalan** [yanue/V2rayU#1254] [Q&A]
这两个都打不开了，现在只有3.2能打开……记忆里之前是3.3的，然后按说明删除了再装那个预发行版，装完直接就弹无法打开了，再下3.3下来装回也打不开了，只有装回3.2才能打开，X86的MAC……<img width="684" alt="image" src="https://github.com/yanue/V2rayU/assets/30583492/ab6aad40-9d34-479b-9fba-2830efb520ae">

**Comment by yysw** [comment]
装3.1.0版可以用
